script/run-local-cy-with-reports.js

```diff
@@ -0,0 +1,152 @@
+/* eslint-disable no-console */
+/* run-local-cy-with-reports.js
+  * Script to run Cypress tests locally with mochawesome reports
+  * Example:
+  * yarn cy:run:localreports "your-spec(s)-glob-pattern"
+  */
+const fs = require('fs');
+const path = require('path');
+const rimraf = require('rimraf');
+const cheerio = require('cheerio');
+const { execSync } = require('child_process');
+const chalk = require('chalk');
+
+const cyLocalReportsHtml = require('./get-cy-local-reports-index');
+
+// Get spec-files parameter from command line arguments
+const spec = process.argv[2] || '';
+console.log(
+  chalk.yellow(`RUNNING CYPRESS WITH LOCAL REPORTS. --spec: ${spec}`),
```

Comment:
Great catch!  I'll add some error-handling here.

Reply:
Great catch!  Error-handling done.  The script will throw error and exit if no app-folder param was found.  Yes, I also refactored the param to take an app-folder instead of a spec path/glob.  Much better now.  Thanks much!